pkg/controllers/status/controller.go

```diff
@@ -602,7 +630,7 @@ func (s *StatusController) realUpdatedReplicas(
 		)
 		if err != nil {
 			keyedLogger.WithValues("cluster-name", clusterName).Error(err, "Failed to get object from cluster")
-			return "", errors.Wrapf(err, "Failed to get %s %q from cluster %q", targetKind, key, clusterName)
+			continue
```

Comment:
@SOF3 same concern here, is it correct to `continue` here?

Reply:
What about just returning error here and letting the caller of `realUpdatedReplicas` log-and-ignore the error?

Actually I think we could even remove this field since we have status aggregator anyway.

Reply:
ok, this PR has been updated, please take a look.